elephant/spike_train_correlation.py

```diff
@@ -906,7 +925,8 @@ def run_T(spiketrain):
 sttc = spike_time_tiling_coefficient
 
 
-def spike_train_timescale(binned_st, tau_max):
+@deprecated_alias(binned_st='binned_spiketrain')
+def spike_train_timescale(binned_spiketrain, tau_max):
```

Comment:
`tau_max` should be `max_tau` per the convention for min/max values. Regarding the Greek letter, this function seems to be specifically tied to the mathematical definition showed in the docstring, so I would leave it as `tau` unless another meaningful name can come.

Reply:
True, but of course, here we again have the stupid case where `\tau_{max}` is a parameter of the function. Since I think the decision was to be consistent in not calling variable names as latex-like mathematical variables, I guess that's good to change...

Reply:
put in the "unsure" section